docs/modules/ROOT/pages/explanations/what_is.adoc

```diff
@@ -0,0 +1,34 @@
+= What is the Application Catalog
+
+TL;DR::
+* A catalog of services which can be self-service ordered directly from a Kubernetes cluster
+* A framework to build services which are then available in the catalog
+
+[quote, products.docs.vshn.ch]
+____
+The VSHN Application Catalog is a Cloud Native marketplace which offers services from Cloud Service Providers like Amazon AWS, Google Cloud, Microsoft Azure, Aiven.io, Exoscale or cloudscale.ch, as well as managed services from VSHN.
+____
+
+Behind the scenes it's *our framework* to self-service deploy applications on Kubernetes, by simply ordering a new instance directly in the cluster (no external system involved).
```

Comment:
> no external system involved

That's not true for resold services. Or what exactly is an "external system" in this context?

Reply:
I rephrased this part as it doesn't make sense when I've read it again.